helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/AvroCompatibilityHelper.java

```diff
@@ -100,6 +115,14 @@ public static AvroVersion getRuntimeAvroVersion() {
     return DETECTED_VERSION;
   }
 
+  /**
+   * returns the detected runtime version of avro-compiler, or null if none found
+   * @return the version of avro detected on the runtime classpath, or null if no avro found
+   */
+  public static AvroVersion getRuntimeAvroCompilerVersion() {
+    return DETECTED_COMPILER_VERSION;
+  }
+
```

Comment:
This is only public in order to test against this method. Not sure if there is a better way without using Guava's `@VisibleForTesting`

Reply:
Good point, I'll leave it as `public`